tasks/interactive-bots.md

```diff
@@ -284,7 +282,7 @@ the `GCI tasks` stream. Consult with mentors and make sure your idea is approved
   * The bot posts the expected output.
 
 * Document your interactive bot at `~/zulip/contrib_bots/lib/<bot_name>` in a
-`docs.md`. Add any relevant screenshots that will help understand how the bot
+`docs.md` (about the bot). Add any relevant screenshots that will help understand how the bot
```

Comment:
So maybe add more clarity? E.g. `(description of how the bot works, example usage etc.)`?